packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -6421,6 +6462,10 @@ class SemanticsConfiguration {
     if (_flags.hasRepeatedFlags(other._flags)) {
       return false;
     }
+    if (_flags.isAccessibilityFocusBlocked != other._flags.isAccessibilityFocusBlocked) {
```

Comment:
if one is BlockSubTree another one is BlockCurrent, I think they can merge to be BlockSubTree

Reply:
added to the absorb